js/packages/openinference-instrumentation-openai/src/instrumentation.ts

```diff
@@ -158,11 +232,11 @@ export class OpenAIInstrumentation extends InstrumentationBase<typeof openai> {
       (original: EmbeddingsCreateType): any => {
         return function patchedEmbeddingCreate(
           this: unknown,
-          ...args: Parameters<typeof module.OpenAI.Embeddings.prototype.create>
+          ...args: Parameters<EmbeddingsCreateType>
         ) {
           const body = args[0];
           const { input } = body;
-          const isStringInput = typeof input == "string";
+          const isStringInput = typeof input === "string";
```

Comment:
oh thanks. Actually feel free to boost our eslint rule on this one.

Reply:
addressed in 31c48d4